python/pyspark/sql/dataframe.py

```diff
@@ -697,6 +697,83 @@ def subtract(self, other):
         """
         return DataFrame(getattr(self._jdf, "except")(other._jdf), self.sql_ctx)
 
+    def dropna(self, how='any', thresh=None, subset=None):
```

Comment:
note to self: add DataFrame.na.fill as alias for DataFrame.fillna, and DataFrame.na.drop as alias for DataFrame.dropna